daras_ai/image_input.py

```
@@ -61,6 +63,56 @@ def resize_img_fit(img_bytes: bytes, size: tuple[int, int]) -> bytes:
+            relpath = Path(furl(url).pathstr).relative_to(settings.MEDIA_URL)
+        except ValueError:
+            return
+        abspath = settings.MEDIA_ROOT / relpath
+        abspath.unlink(missing_ok=True)
+
+
+@contextmanager
+def temp_upload_file_from_bytes(
+    filename: str,
+    data: bytes,
+    content_type: str = None,
```

Ruff (0.15.17)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)